tests/create.pp

```diff
@@ -83,3 +83,19 @@
     port     => 80,
   }],
 }
+
+rds_instance { 'db-name-5':
```

Comment:
Something like:

Create RDS MySQL instance, connect to instance using puppetlabs-mysql?